cardano-cli/src/Cardano/CLI/Options.hs

```diff
@@ -60,6 +61,13 @@ addressCmdsTopLevel envCli = AddressCommand <$> pAddressCmds envCli
 nodeCmdsTopLevel :: Parser ClientCommand
 nodeCmdsTopLevel = NodeCommands <$> pNodeCmds
 
+-- Queries actually depend on the node to client version which may coincide
+-- with a hardfork but not necessarily. We will expose commands at the top level
```

Comment:
>We will expose commands at the top level (...)

>The help text should be (...)

I don't think mentioning our intentions in the comments brings any value. Unless it's a TODO, which is easier to spot and correct than a prose with wishes.